xinference/client.py

```diff
@@ -480,6 +480,24 @@ def _gen_model_uid(cls) -> str:
         # generate a time-based uuid.
         return str(uuid.uuid1())
 
+    def register_model(self, model_type: str, model: str, persist: bool):
```

Comment:
Can you give a test sample of `register_model`?

Reply:
Done.